src/components/Navigation/Navigation.module.css

```diff
@@ -0,0 +1,40 @@
+.nav {
+  display: flex;
+  flex-direction: column-reverse;
+  row-gap: 32px;
+  position: fixed;
+  top: 40px;
+  left: 20px;
+  right: 20px;
+  pointer-events: all;
+  z-index: 5;
```

Comment:
Always scary, is it required? Especially as `fixed` would usually put it on top already

Reply:
Felt better to be sure, it needs to cover everything. Happy to look at removing it.